.gitignore

```diff
@@ -1,12 +1,13 @@
 node_modules
 dist
+.env
+.aider*
+prisma/
```

Comment:
This is probably wrong. We want to keep `prisma/migrations/*`, `prisma/schema.prisma` and more in version control.

`prisma/generated` could be gitignored, but the rest should be kept in git.